examples/regression.py

```diff
@@ -3,18 +3,25 @@
 import functorch
 from functorch._src.compile_utils import strip_overloads
 import iree_torch
-import numpy as np
-from sklearn.datasets import make_regression
-from sklearn.model_selection import train_test_split
 import torch
 import torch_mlir
 
 
 # Dataset
-X, y, coef = make_regression(n_features=3, coef=True)
-X = torch.from_numpy(X).to(dtype=torch.float32)
-y = torch.from_numpy(y).to(dtype=torch.float32)
-X, X_test, y, y_test = train_test_split(X, y)
+X = torch.tensor((
```

Comment:
It should be possible to generate this with a handful of lines of code. You can generate a holdout target weight/bias matrix (not seen by the training procedure), and then run random inputs through it and add some noise.

Reply:
Done.